airbyte-ci/connectors/metadata_service/lib/tests/fixtures/metadata_validate/valid/with_optional_field/with_releases/metadata_breaking_changes_with_impact_scopes.yaml

```diff
@@ -0,0 +1,22 @@
+metadataSpecVersion: 1.0
+data:
+  name: AlloyDB for PostgreSQL
+  definitionId: 1fa90628-2b9e-11ed-a261-0242ac120002
+  connectorType: source
+  dockerRepository: airbyte/image-exists-1
+  githubIssueLabel: source-alloydb-strict-encrypt
+  dockerImageTag: 0.0.1
+  documentationUrl: https://docs.airbyte.com/integrations/sources/existingsource
+  connectorSubtype: database
+  releaseStage: generally_available
+  license: MIT
+  releases:
+    breakingChanges:
+      2.0.0:
+        upgradeDeadline: 2023-08-22
+        message: "This version changes the connector’s authentication method from `ApiKey` to `oAuth`, per the [API guide](https://amazon-sqs.com/api/someguide)."
+        scopedImpact:
+          - scopeType: stream
+            impactedScopes: ["affected_stream"]
```

Comment:
Shall we also add a test to check that multiple entries under `scopedImpact` is valid?
```yaml
scopedImpact:
  - scopeType: stream
     impactedScopes: ["affected_stream", "foo"]
  - scopeType: stream
     impactedScopes: ["another_affected_stream"]
```

Reply:
Yep - done! [8ffe17d](https://github.com/airbytehq/airbyte/pull/33111/commits/8ffe17dc7878cbf6507724ec98b048dd1a2f47f3)